packages/flutter/lib/src/rendering/viewport.dart

```diff
@@ -783,7 +783,6 @@ abstract class RenderViewportBase<ParentDataClass extends ContainerParentDataMix
     final double offsetDifference = offset.pixels - targetOffset;
 
     final Matrix4 transform = target.getTransformTo(this);
-    applyPaintTransform(child, transform);
```

Comment:
It seems the paint transform that does `child -> this` is applied twice here.

Reply:
Nice catch. Did you add a test for this?

Reply:
Removed. Adding this in a different PR.